src/App.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable import/extensions */
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import Navigation from './layouts/navigation';
+import Rockets from './components/Rockets/Rockets';
+import Missions from './components/missions/missions';
+import MyProfile from './components/myProfile/myProfile';
+import store from './redux/configureStore';
+
+function App() {
+  return (
+    <>
+      <Provider store={store}>
```

Comment:
- [ ] When I start the server, I get complaints on the terminal. Kindly check it out and fix it, it may be a naming issue.

![image](https://user-images.githubusercontent.com/4247740/168440889-e81a0ff1-be90-45fb-b35d-9b0c1b36db81.png)